webpack/assets/javascripts/react_app/components/common/AutoComplete/index.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import Downshift from 'downshift';
+import classnames from 'classnames';
+import { InputGroup } from 'react-bootstrap';
+import { Button } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+import './AutoComplete.scss';
+
+export const renderItems = ({
+  items, highlightedIndex, selectedItem, getItemProps,
+}) => (
+  <div className="autocomplete-dropdown-container">
+    {items.map((item, index) => (
+      <div
+        className={classnames({
+          'autocomplete-dropdown-item': true,
+          highlighted: highlightedIndex === index,
+          selected: selectedItem === item,
+        })}
+        {...getItemProps({
+          key: item,
+          index,
+          item,
+        })}
+      >
+        {item}
+      </div>
+    ))}
+  </div>
+);
+
+const AutoComplete = ({
+  onSelect, labelText, onInputUpdate, items,
+}) => (
+  <Downshift
+    onChange={onSelect}
+    itemToString={i => (i == null ? '' : String(i))}
+    render={({
+      getInputProps,
+      getItemProps,
+      getLabelProps,
+      isOpen,
+      inputValue,
+      highlightedIndex,
+      getRootProps,
+      selectedItem,
+    }) => (
+      <div>
+        {labelText ? <label {...getLabelProps()}>{labelText}</label> : null}
+        <InputGroup>
+          <input
```

Comment:
You can build your form with `patternfly-react` components only:
https://rawgit.com/patternfly/patternfly-react/gh-pages/index.html?knob-Disabled=false&knob-Style=default&knob-Show%20Labels=false&knob-Show%20Help=true&knob-Show%20Loading=false&selectedKind=Forms&selectedStory=Supported%20Controls&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs